.gitignore

```diff
@@ -1,3 +1,8 @@
 build
 .cache
 *.DS_Store
+node_modules
+site-local.yml
+package-lock.json
+package.json
```

Comment:
Why did you add the `package.json` file here? Doesn't it define the required NPM packages which should be the same for everyone?

Reply:
It wasn't there before, but you're right, it does not belong in .gitignore, removed.